server/loginserver.ts

```diff
@@ -73,27 +71,26 @@ class LoginServerInstance {
 			}
 		}
 
-		const actionUrl = url.parse(this.uri + 'action.php' +
+		const actionUrl = url.parse(
```

Comment:
`url.parse` shouldn't be necessary, actually. It's only used to parse the URI for use by `http.get`/`https.get`, which is already done inside `Net`.